pages/index.vue

```diff
@@ -1,60 +1,13 @@
+<template>
+  <TodoApp />
+</template>
 <script>
-import { mapMutations } from 'vuex'
+import TodoApp from '../components/todo/Index.vue'
```

Comment:
We could use `@` here instead of `../` to keep the import consistant across components (easier for refactoring afterwards).

Reply:
Yes, bhai, noted, I'll use the relative path afterwards. Thanks